documentatie/flowcharts/data-entry-forms.md

```diff
@@ -153,67 +148,73 @@ flowchart TD
     user-addresses-error -- resolve --> change-input
-    change-input --> click-next
+    change-input --> next-button
 
     error-cur-page -- no --> warning-cur-page
     warning-cur-page -- yes --> warnings-accepted
     warnings-accepted -- no --> user-addresses-warning
     warnings-accepted -- yes --> go-to-next-page
     user-addresses-warning -- resolve --> change-input
     user-addresses-warning -- accept --> accept-warning
-    accept-warning --> click-next
+    accept-warning --> next-button
     user-addresses-warning -- abort --> abort-input
 
     warning-cur-page -- no --> go-to-next-page
-
 ```
 
-Note that if there is a warning and the user changes the input, they should no longer have the option to accept the warning. They need to click "Next" first, to validate the changed input.
+- If there is a warning and the user changes the input, they should no longer have the option to accept the warning. They need to click "Next" first, to validate the changed input.
+- Currently, only error F.204 can trigger on a previous page such that the user is redirected to that page.
+
+## Navigate away from page
 
+Navigating away from a page can happen in several ways:
+- Clicking an item in the left navigation menu
+- Clicking a link in an error or warning message
+- Clicking a link in the top navigation bar
+- Clicking the browser back/forward button
 
-## Click navigation item or browser back/forward buttons
+The next page can be either within the form or outside it. The action flow depends on this.
 
 ```mermaid
 flowchart TD
     %% start and end
-    flow-start([start])
-
+    flow-start([navigate action])
     go-to-page([go to selected page])
     remain-on-page([remain on current page])
 
     %% steps
-    click-nav-item(click navigation item)
-    click-browser-back("click browser
-        back button")
-    click-browser-forward("click browser
-        forward button")
+    inside-outside{navigating inside form?}
     on-furthest-page{on furthest page?}
     user-made-changes{user made changes?}
     save-changes{save changes?}
-    call-api(call api)
+    call-save-api(call save api)
+    call-delete-api(call delete api)
     cache-input(cache input)
     reset-changes(reset changes)
     errors-or-warnings{errors or warnings?}
+    modal-fix-or-ignore{fix or ignore?}
+    save-or-delete{"save or delete?"}
 
     %% flow
-    flow-start --> click-nav-item
-    flow-start --> click-browser-back
-    flow-start --> click-browser-forward
-    click-browser-back --> on-furthest-page
-    %% if you can click the browser forward button, you by definition are not on the furthest page
-    click-browser-forward --> user-made-changes
-    click-nav-item --> on-furthest-page
+    flow-start --> inside-outside
+    inside-outside -- outside --> save-or-delete
+    inside-outside -- inside --> on-furthest-page
     on-furthest-page -- yes --> cache-input
     cache-input --> go-to-page
+    
+    save-or-delete -- save --> call-save-api
```

Comment:
This part of the flow seems too restrictive. If you want to navigate out of the data entry of a polling station without deleting all the data entry you have done so far, your only option is to save what's in the current form. Even if you want to revert to the latest input you actually submitted. And if there's an error or warning for what's in the current form, you're not allowed to navigate away unless you fix those.

We can revisit this when we pick up epic #95. For this PR I'd leave this part as-is.

Reply:
This is changed now to be the same modal as navigating inside the form. Only when explicitly clicking the 'abort data entry' button we will have  the option to delete.